docs/content/users/usage/cli.md

```diff
@@ -92,12 +92,12 @@ ddev snapshot
 Creating database snapshot d9_20220107124831-mariadb_10.3.gz
 Created database snapshot d9_20220107124831-mariadb_10.3.gz
 
-ddev snapshot restore d9_20220107124831-mariadb_10.3.gz
+ddev snapshot restore d9_20220107124831
 Stopping db container for snapshot restore of 'd9_20220107124831-mariadb_10.3.gz'...
 Restored database snapshot d9_20220107124831-mariadb_10.3.gz
 ```
 
-Snapshots are stored as gzipped files in the project’s `.ddev/db_snapshots` directory, and the file created for a snapshot can be renamed as necessary. For example, if you rename the above `d9_20220107124831-mariadb_10.3.gz` file to `working-before-migration-mariadb_10.3.gz`, then you can use `ddev snapshot restore working-before-migration-mariadb_10.3.gz`. (The description of the database type and version—`mariadb_10.3`, for example—must remain intact.)
+Snapshots are stored as gzipped files in the project’s `.ddev/db_snapshots` directory, and the file created for a snapshot can be renamed as necessary. For example, if you rename the above `d9_20220107124831-mariadb_10.3.gz` file to `working-before-migration-mariadb_10.3.gz`, then you can use `ddev snapshot restore working-before-migration`. (The description of the database type and version — `mariadb_10.3`, for example — must remain intact.)
```

Comment:
Looks like you specifically added spaces around the em dash. Apparently Chicago and AP Style are split on this, but in the docs we’ve consistently gone with the spaceless variant. Would you mind changing it back or is there a compelling reason we should adopt this style?

Reply:
@rfay My account got flagged (39 days! And they even didn't tell me why it was flagged...), therefore all my contributions were hidden.

@mattstein Oh, I was not aware that there are rules for this (english is not my mother tongue). I just added them because it looked falsy. But of course can I change it back. I will push an update today.

Reply:
That explains why I didn't see it! Thanks for letting me know.

Reply:
Thank you @rintisch!